rasa/.vscode/launch.json

```diff
@@ -7,7 +7,10 @@
             "request": "launch",
             "module": "rasa",
             "cwd": "${env:BF_PROJECT_PATH}",
-            "args": ["run", "--debug", "--enable-api"]
-        }
-    ]
+            "args": ["run", "--debug", "--enable-api"],
+            "env": {
+                "BF_PROJECT_ID": "QAep494JdyvyvrpnN",
```

Comment:
should the default Id be bf? (not super imortant)

Reply:
Well it doesn't default to bf anynore, so I though it doesn't matter, and I think it looks more like something that has to be changed this way